types.d.ts

```diff
@@ -400,6 +400,225 @@ declare class Logger {
     emergency: Log;
 }
 
+/**
+ * MMRP Envelopes are used to encapsulate data to
+ * be sent between MAGE nodes.
+ *
+ * @class MmrpEnvelope
+ */
+declare class MmrpEnvelope {
+    constructor(eventName: string, messages: string | Buffer | string[] | Buffer[], route: string[], returnRoute: string[], flags: any);
```

Comment:
NM, found it in the API doc; however, what would happen if I specified both NONE and TRACK_ROUTE in an array?